picspace/src/Components/RandomPictures.js

```diff
@@ -1,21 +1,63 @@
 import React from "react";
+import axios from 'axios';
 import './RandomPictures.css';
 
-const RandomPicture = ({randomPictures}) => {
-  return (
-    <article className="RandomPictures">
+class RandomPictures extends React.Component {
+  constructor(props){
+    super(props)
+    this.state = {
+      id: "",
+      name : "",
+      file_url: "",
+      arrayId: [],
+      arrayPictures:[],
+    }
+  this.getIdPicture = this.getIdPicture.bind(this);
+  }
+ 
+  getIdPicture() {
+    var tempArray = [];
```

Comment:
No more var

Reply:
i changed the type of variable (var > let).
i used ES6 synthax for url.